lib/puppet_metadata/github_actions.rb

```diff
@@ -62,6 +62,7 @@ def github_action_test_matrix(use_fqdn: false, pidfile_workaround: false)
             next unless setfile
 
             matrix_include << {
+              name: "#{puppet_version[:name]} - #{setfile[1]}",
```

Comment:
With the new reusable action, it may be sufficient to use just the version
```suggestion
              name: "#{puppet_version[:value]} - #{setfile[1]}",
```